storage/src/main/java/org/apache/kafka/storage/internals/log/LogFileUtils.java

```diff
@@ -16,7 +16,12 @@
  */
 package org.apache.kafka.storage.internals.log;
 
+import java.io.File;
+import java.text.NumberFormat;
+
 public final class LogFileUtils {
+    public static final String PRODUCER_SNAPSHOT_FILE_SUFFIX = ".snapshot";
+    public static final String DELETED_FILE_SUFFIX = ".deleted";
```

Comment:
Worth adding the comment from `LocalLog`:

`/** a file that is scheduled to be deleted */`